modules/security/src/main/java/org/eclipse/xpanse/modules/security/config/AesUtil.java

```diff
@@ -0,0 +1,120 @@
+/*
+ * SPDX-License-Identifier: Apache-2.0
+ * SPDX-FileCopyrightText: Huawei Inc.
+ *
+ */
+
+package org.eclipse.xpanse.modules.security.config;
+
+import java.io.File;
+import java.io.FileInputStream;
+import java.io.IOException;
+import java.io.InputStream;
+import java.nio.charset.StandardCharsets;
+import java.security.InvalidAlgorithmParameterException;
+import java.security.InvalidKeyException;
+import java.security.NoSuchAlgorithmException;
+import java.util.Objects;
+import javax.crypto.BadPaddingException;
+import javax.crypto.Cipher;
+import javax.crypto.IllegalBlockSizeException;
+import javax.crypto.NoSuchPaddingException;
+import javax.crypto.SecretKey;
+import javax.crypto.spec.IvParameterSpec;
+import javax.crypto.spec.SecretKeySpec;
+import lombok.extern.slf4j.Slf4j;
+import org.apache.commons.codec.binary.Base64;
+import org.eclipse.xpanse.modules.models.service.deploy.exceptions.SensitiveFieldEncrytionOrDecryptionFailedException;
+import org.springframework.stereotype.Component;
+
+/**
+ * AES encryption tool class.
+ */
+@Slf4j
+@Component
+public class AesUtil {
+
+    private static final String AES = "AES";
+    private static final String VI = "c558Gq0YQK2QUlMc";
+    private static final String AES_KEY_FILE_NAME = "aes_sec";
```

Comment:
also add aes_sec file to .gitignore so that no one checks it in to repo by mistake.